metadata/release/sample.json

```diff
@@ -40,56 +122,72 @@
                 "sha256": "881178a4794816e623b02012a84b11d59a96dd59035508a0986a5b6c6be074ed"
               }
             },
-            "installer.iso": {
+            "live.iso": {
```

Comment:
This doesn't match the [actual release metadata](https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220505.3.2/release.json), which just calls it `iso`.  Similarly for `pxe` below.